packs/chatops_tests/actions/test_aliases_with_slack.py

```diff
@@ -144,12 +144,18 @@ def test_help_shortcut(self):
             if filtered_messages:
                 messages.extend(filtered_messages)
 
-        self.assertEqual(1, len(messages))
-        if len(messages) != 1:
+        self.assertEqual(2, len(messages))
+        if len(messages) != 2:
             time.sleep(self.WAIT_FOR_MESSAGES_TIMEOUT)
 
+        # Help commands should returns more than 100
+        commands_len = 0
+        for message in messages:
+            commands_len = commands_len + len(message['text'].split('\n'))
```

Comment:
Let's constrain this a bit and make sure that we are getting exactly the number of `[unused]` aliases we are expecting to get.

You'll need to combine the text from both messages, then split them on `\n`, and then filter out any that don't have `[unused]` in them.

```python
combined_text = messages[0]['text'] + messages[1]['text']
number_of_unused_commands = len(filter(lambda line: line.startswith('[unused']), combined_text.split('\n')))
```

You might need to play around with the value in `.startswith()` to select the correct aliases. And you may need to use a regex (but avoid using regexes if you can, since they are much slower than simple string functions).